chainer/links/connection/inceptionbn.py

```diff
@@ -33,6 +35,8 @@ class InceptionBN(link.Chain):
             ``cupy.ndarray`` and edits its value.
             It is used for initialization of the convolution matrix weights.
             Maybe be ``None`` to use default initialization.
+        dtype (numpy.dtype): Type to use in
+            ``~batch_normalization.BatchNormalization``.
```

Comment:
Why ``dtype`` is only used in ``BatchNormalization`` layers, and not used in convolutional layers?